packages/elements-core/src/components/Docs/Security/PanelContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { MarkdownViewer } from '@stoplight/markdown-viewer';
+import { Box, NodeAnnotation } from '@stoplight/mosaic';
+import { HttpSecurityScheme } from '@stoplight/types';
+import * as React from 'react';
+
+import { useOptionsCtx } from '../../../context/Options';
+import { getReadableSecurityName, shouldIncludeKey } from '../../../utils/oas/security';
+import { getDefaultDescription } from '../../../utils/securitySchemes';
+
+export const PanelContent: React.FC<{ schemes: HttpSecurityScheme[] }> = ({ schemes }) => {
+  const { nodeHasChanged } = useOptionsCtx();
+
+  if (!schemes || !schemes.length) {
+    return null;
+  }
+
+  const collection = schemes.length > 1;
+
+  return (
+    <Box>
+      {schemes.map(scheme => (
+        <Box key={scheme.key} p={2} m={2} border>
+          {collection && (
+            <MarkdownViewer
```

Comment:
I forgot to mention it in in the other PR but using markdown viewer to render a plain header is an overkill.
IIRC we have `Heading` or something like that, or we could just use `Text`. There's isn't much happening here anyway

Reply:
Don't feel obliged to make that change - I don't think it's a new code so it's alright to keep it as is.